modules/preview-react/divider/lib/Divider.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {createStencil, handleCsProp, CSProps, calc, cssVar} from '@workday/canvas-kit-styling';
+import {system} from '@workday/canvas-tokens-web';
+
+export const dividerStencil = createStencil({
+  vars: {
+    space: cssVar(system.space.x4),
+  },
+  base: ({space}) => {
+    return {
+      display: 'block',
+      height: 1,
+      border: 'none',
+      borderTop: `1px solid ${system.color.border.divider}`,
+      margin: `${calc.divide(space, 2)} 0`,
+    };
+  },
+});
+
+interface DividerProps extends CSProps {
+  space?: string;
```

Comment:
`space` is a convenient way to adjust the vertical margin without having to adjust it with `createStyles`.

Reply:
Thoughts: Do we outline/call  that this is `verticalSpacing`?

Reply:
It would be nice if in the future this component could have a vertical orientation.
```ts
<Divider orientation="vertical" space="1rem" />
```

In which case, the `space` would be horizontal.